test/clj/rems/api/test_email.clj

```diff
@@ -4,14 +4,14 @@
             [rems.api.testing :refer :all]
             [rems.db.outbox :as outbox]
             [rems.db.test-data-helpers :as test-helpers]
+            [rems.db.test-data :as test-data]
             [rems.handler :refer [handler]]
             [ring.mock.request :refer :all]))
 
 (use-fixtures
-  :once
-  api-fixture)
+  :each
+  api-fixture-without-data)
 
-(def api-key "42")
 (def user-id "developer")
```

Comment:
Same here, the constant does not seem useful to us.

Reply:
done